BlockchainSdk/Extensions/String+.swift

```diff
@@ -8,19 +8,33 @@
 
 import Foundation
 
+fileprivate var hexPrefix = "0x"
+
 extension String {
     func contains(_ string: String, ignoreCase: Bool = true) -> Bool {
         return self.range(of: string, options: ignoreCase ? .caseInsensitive : []) != nil
     }
-    
-    func removeHexPrefix() -> String {
-        if self.lowercased().starts(with: "0x") {
+
+    public func hasHexPrefix() -> Bool {
+        return self.lowercased().hasPrefix(hexPrefix)
+    }
+
+    public func removeHexPrefix() -> String {
+        if hasHexPrefix() {
             return String(self[self.index(self.startIndex, offsetBy: 2)...])
```

Comment:
NP: А здесь может переделать тут
```suggestion
            return String(dropLast(2))
```

Reply:
только dropFirst(2) 😀, сделал